tools/Localizations/Sources/Localizations/main.swift

```diff
@@ -75,13 +75,53 @@ struct L10NTools: ParsableCommand {
         default: return true;
         }
     }
-    
+
+    private func getLocalesFromProjectFolder () -> (Array<String>) {
+        var myLocalesList:[String] = []
+
+        let blockzillaFolder = FileManager.default.enumerator(atPath: URL(fileURLWithPath: projectPath).deletingLastPathComponent().appendingPathComponent("Blockzilla").path)
+
+        let filePaths = blockzillaFolder?.allObjects as! [String]
+        let textFilePaths = filePaths.filter{$0.contains(".lproj")}
+        for txt in textFilePaths {
+            if let index = txt.firstIndex(of: ".") {
+                let firstPart = txt.prefix(upTo: index)
+                myLocalesList.append(String(firstPart))
+            }
+        }
+        var uniqueLocales = Array(Set(myLocalesList))
```

Comment:
Let's add a few comments here to explain what it happening. I think you need to unique the locales because the `enumerator()` will find multiple lproj bundles for the same locale?